src/server/tb/home/index.js

```diff
@@ -0,0 +1,24 @@
+/** @import { ServerRegisterPluginObject } from '@hapi/hapi' */
+
+import { PageController } from '~/src/server/common/controller/page-controller/page-controller.js'
+import { Page } from '~/src/server/common/model/page/page-model.js'
+import { config } from '~/src/config/config.js'
+
+const pageHeadingAndTitle =
+  'Apply for a Bovine Tuberculosis (TB) movement licence'
+
+export class HomePage extends Page {
+  key = 'home'
+  sectionKey = 'tb'
+  view = 'tb/home/index.njk'
+  urlPath = '/'
+  pageTitle = config.get('serviceName')
```

Comment:
Title and heading should match, so maybe: 

```
const pageHeadingAndTitle = config.get('serviceName')

...

  pageTitle = pageHeadingAndTitle
  pageHeading = pageHeadingAndTitle

```

Also notice that we've got a property - pageHeading - and a getter `heading` in the base page model, which seems redundant.  Not 100% sure why we've got that 🤔

Reply:
using heading in the template so both are not the same now